### PR TITLE
Revise 'getting started' and rename as 'overview'

### PR DESCRIPTION
**Pull request by jrmhaig**

### CODE REVIEW

File: _documentation/01-overview.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+### Hive Runner
+
+The Hive Runner, or Hive, is the component that executes the tests and submits
+results. It is also responsible for monitoring and looking after devices.
+
+The core of the Hive is the `hive-runner` gem and it is configured with one or
+more runner plugins depending on the device types that are to be connected.
+Currently there are plugins for:
+
+* Android mobiles devices, from the `hive-runner-android` gem
```
Comment (jonpwilson) [inline _documentation/01-overview.md]: Android mobile**S** devices

File: _documentation/01-overview.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+Both Hive Scheduler and Hive Mind are web services that provide a web interface
+and an API for communicating with devices and hives. The Hive Runners, normally
+referred to simply as 'Hives' are servers to which the devices are connected.
+These may be located at multiple locations provided they are able to have a
+network connection to Hive Schduler and Hive Mind.
+
+### Hive Scheduler
+
+The scheduler is the main hive application. It contains the definitions of your
+tests and allows you to scheduler those tests to run on devices that are connected
```
Comment (jonpwilson) [inline _documentation/01-overview.md]: allows you to schedule**r**

Comment (jonpwilson) [inline _documentation/01-overview.md]: Also - this isn't technically true as you can't schedule anything

File: _documentation/01-overview.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+The Hive Runner, or Hive, is the component that executes the tests and submits
+results. It is also responsible for monitoring and looking after devices.
+
+The core of the Hive is the `hive-runner` gem and it is configured with one or
+more runner plugins depending on the device types that are to be connected.
+Currently there are plugins for:
+
+* Android mobiles devices, from the `hive-runner-android` gem
+* iOS mobiles devices, from the `hive-runner-ios` gem
```
Comment (jonpwilson) [inline _documentation/01-overview.md]: mobiles again

File: _documentation/01-overview.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+The complete setup containing these three components looks like this:
+
+![Hive CI overview](/hive-ci/images/full-hive-ci.png){: class="img-responsive" }
+
+Both Hive Scheduler and Hive Mind are web services that provide a web interface
+and an API for communicating with devices and hives. The Hive Runners, normally
+referred to simply as 'Hives' are servers to which the devices are connected.
+These may be located at multiple locations provided they are able to have a
+network connection to Hive Schduler and Hive Mind.
```
Comment (Asimk21) [inline _documentation/01-overview.md]: Scheduler

File: _documentation/01-overview.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+![Hive Runner top level](/hive-ci/images/hive-runner-and-plugin-details.png){: class="img-responsive" }
+
+When you install the runner, you will be guided to install additional gems to
+support the devices you want to test.
+
+### HiveMind
+
+Hive Mind is a database application for monitoring devices and device state. This
+project isn't yet ready for open source release, but you can still set up your
```
Comment (jonpwilson) [inline _documentation/01-overview.md]: It is open source!

**Pull request merged**